package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "carbon-factory",
-  "version": "8.0.2",
+  "version": "8.0.3",
```

Comment:
```suggestion
  "version": "8.1.0",
```
@edleeks87 IMO when bumping versions, we should increment the highest dependency that we have changed in case any of our consumers are depending upon a version range.

I tested `carbon` master with this version of `carbon-factory` and everything works as expected and the vulnerabilities have been resolved.

Thank you! 😄

Reply:
Yeah agreed, this was a hangover from the original update of just the lock file so was ok as just a patch, I'll update the package files and push it up again